ironfish-cli/src/commands/wallet/prune.ts

```diff
@@ -71,6 +71,10 @@ export default class PruneCommand extends IronfishCommand {
       CliUx.ux.action.stop()
     }
 
+    CliUx.ux.action.start(`Cleaning up deleted accounts`)
+    await node.wallet.forceCleanupDeletedAccounts()
```

Comment:
We should probably do this before the call to `compact`, above, to get the most we can out of compaction